Specs/Renderer/TextureSpec.js

```diff
@@ -1266,14 +1270,14 @@ describe(
     });
 
     it("throws when copying to a texture from the framebuffer with a compressed pixel format", function () {
-      if (context.s3tc) {
+      if (context.s3tc || context.pvrtc || context.etc1) {
```

Comment:
Maybe use `FeatureDetection.supportsBasis` instead?

Reply:
We'd need a new `import { FeatureDetection } from "../../Source/Core/FeatureDetection.js";` added above. is that fine?

Reply:
Yes, that's fine